app/components/endpointInit/endpointInitController.js

```diff
@@ -48,9 +48,10 @@ function ($scope, $state, EndpointService, StateManager, EndpointProvider, Notif
     $scope.state.error = '';
     var name = "local";
     var URL = "unix:///var/run/docker.sock";
+    var URLPublish = "_";
```

Comment:
I do not like this workaround here (and the one in the containers view to determine if the URLPublish is defined). Update the API to allow URLPublish to be undefined (see my first comment).